src/components/common/form/error-summary/ErrorSummary.jsx

```diff
@@ -2,6 +2,18 @@ import PropTypes from 'prop-types';
 import { HashLink } from 'react-router-hash-link';
 
 const ErrorSummary = ({ errors }) => {
+  const callbackRef = (node) => {
+    // if node exists (there is a summary error), scroll to the first summary error and focus it
+    if (node) {
+      node.children['summary-error-0'].children[
+        'summary-error-0-message'
```

Comment:
Could this be put into a variable so that we don't need the comment? Maybe something like `firstSummaryError`?

Reply:
Good call, I've changed the element selection to a separate variable